projects/demo/src/app/app.component.ts

```diff
@@ -95,9 +98,15 @@ export class AppComponent implements OnInit, OnDestroy {
     { name: 'RESOURCES', children: [] },
   ];
 
-  public darkMode = new FormControl<boolean>(false);
+  // Use a string FormControl to hold the segmented control value ('light-mode' | 'dark-mode')
+  public darkMode = new FormControl<string>('light-mode');
```

Comment:
Firstly, darkMode variable holds string values (`'light-mode' | 'dark-mode'`) values, may a different name would be more feasible, like `theme` or something like that. 
Secondly, the formControl type can be `'light-mode' | 'dark-mode'` as you commented